src/dry-run.ts

```diff
@@ -43,7 +43,7 @@ async function check (args: any): Promise<void> {
     if (commitMessage) {
       console.log('This appears to be a valid Dependabot Pull Request.')
 
-      const updatedDependencies = parse(commitMessage)
+      const updatedDependencies = parse(commitMessage, 'dependabot/test_ecosystem/test_directory', 'test_branch')
```

Comment:
lets use `getBranchNames` here since we have access to the GitHub `Context` actually pass in the `headName` and `baseName`

Reply:
Good catch - updated.

Reply:
I had to add another rest call to get the actual pull_request data since the `actionContext` is faked out in `dry-run.ts`.

This should now work correctly.